docs/advanced-guide/using-publisher-subscriber/page.md

```diff
@@ -27,10 +27,23 @@ that are specific for the type of message broker user wants to use.
 ### KAFKA
 
 #### Configs
+| Name                  | Description                                                                                                            | Required     | Default   | Example          | Valid format     |
+|-----------------------|------------------------------------------------------------------------------------------------------------------------|--------------|-----------|------------------|------------------|
+| `PUBSUB_BACKEND`      | Using Apache Kafka as message broker                                                                                   | +            |           | `KAFKA`          | Not empty string |
+| `PUBSUB_BROKER`       | Address to connect to kafka broker                                                                                     | +            |           | `localhost:9092` | Not empty string |
+| `CONSUMER_ID`         | Consumer group id to uniquely identify the consumer group                                                              | if consuming |           | `order-consumer` | Not empty string |
+| `PUBSUB_OFFSET`       | Determines from whence the consumer group should begin consuming when it finds a partition without a committed offset. | -            | `-1`      | `10`             | int              |
+| `KAFKA_BATCH_SIZE`    | Limit on how many messages will be buffered before being sent to a partition.                                          | -            | `100`     | `10`             | Positive int     |
+| `KAFKA_BATCH_BYTES`   | Limit the maximum size of a request in bytes before being sent to a partition.                                         | -            | `1048576` | `65536`          | Positive int     |
```

Comment:
Any limit on the max batch_bytes if so can be added there

Reply:
I don't think it make sense, in fact all configs have max limit by value type:

* The maximum value for batch.size in Kafka is 2147483647 bytes, which is equivalent to 2^31 - 1 bytes. This is because the batch.size configuration is an integer, and the maximum value for an integer in Java (which Kafka is written in) is 2^31 - 1. 
* The linger.ms configuration in Kafka is a long type, and the maximum value for a long in Java (which Kafka is written in) is 9223372036854775807 or 2^63 - 1. However, this value is in milliseconds, which translates to a very long time (approximately 292 million years). In practical terms, you would typically set linger.ms to a much smaller value, such as a few milliseconds to a few seconds, depending on your specific use case and throughput-latency tradeoff.